### DISCUSSION TITLE
Communication channel

### DISCUSSION THREAD

**Discussion by diananova** [LesFruitsDefendus/saskatoon-ng#205] [Outreachy]
Hi! I'm Diana, an Outreachy applicant 👋. 
I really like the impact this project has in real life and would love to contribute to its mission. 
I noticed that you mostly use Github for communication, have you ever considered using something like Slack/Zulip?

**Comment by tristanlatr** [comment]
Hi @diananova and thank you for your interest into Saskatoon!

We communicate with GitHub for the development, if you’d rather send a private message, you can email us at info@lesfruitsdefendus.org.

The core member of the collective have a private email list in which we communicate. But right now it’s pretty calm.

I’m unsure to see the short term benefits to adopt such a platform, I’ve used Mattermost for various projets, but I don’t think it would fit our usage because there aren’t as many people taking actually. And for the development flow, GitHub seem to work as expected. 

Tell me if it answers the question,

Talk to you later,
